src/lua/LuaShip.cpp

```diff
@@ -660,6 +662,42 @@ static int l_ship_is_ecm_ready(lua_State *l)
 	return 1;
 }
 
+/*
+ * Method: GetDurationForDistance
+ *
+ *   Calculating the duration of the flight of a given ship at a specified distance.
+ *   Assumed that at the beginning and at the end of the travel the speed is 0.
+ *
+ * > duration = ship:GetDurationForDistancel(distance)
```

Comment:
Very minor nitpick but the code example here has a typo. :smile:

Reply:
Done!